src/credentials/attestation_verifier/DeviceAttestationVerifier.h

```diff
@@ -333,6 +333,28 @@ class DeviceAttestationVerifier
     typedef void (*OnAttestationInformationVerification)(void * context, const AttestationInfo & info,
                                                          AttestationVerificationResult result);
 
+    /**
+     * @brief Interface for checking the device attestation revocation status
+     *
+     */
+    class DeviceAttestationRevocationDelegate
```

Comment:
So this is an inner class, not a subclass.  The issue being fixed talks about subclass, no?  What is the actual thing being fixed here?

Reply:
I misunderstood the inner class for subclass. Had a discussion with Tennessee, and this change is not required. Closed the related issue and this PR.